Libraries/StyleSheet/StyleSheet.js

```diff
@@ -160,8 +160,8 @@ module.exports = {
   /**
    * Creates a StyleSheet style reference from the given object.
    */
-  create(obj: {[key: string]: any}): {[key: string]: number} {
-    var result = {};
+  create<T: Object, U>(obj: T): {[key:$Keys<T>]: number} {
+    var result: T = (({}: any): T);
```

Comment:
I think `result` should be of type `{[key: $Keys<T>]: number}`, this checks out on the Try Flow website:

```js
function create<T: Object>(obj: T): {[key: $Keys<T>]: number} {
  var result: {[key: $Keys<T>]: number} = {};
  for (var key in obj) {
    StyleSheetValidation.validateStyle(key, obj);
    result[key] = ReactNativePropRegistry.register(obj[key]);
  }
  return result;
}
```